collect_app/src/main/res/values-af/strings.xml

```diff
@@ -258,4 +257,5 @@
   <!--%1$s represents degrees, %2$s represents minutes and %3$s represents seconds as in S 18°48'8"-->
   <!--%1$s represents degrees, %2$s represents minutes and %3$s represents seconds as in N 18°48'8"-->
   <!--Displays dates in specialized calendars with the Gregorian dates in parentheses. This is translated so right-to-left languages can specify the correct order-->
+  <!--Sms submission-->
```

Comment:
All of these are auto-generated from the English file. There was a comment added there so it was added here. We just let those go through.

Reply:
Just wanted to make sure that it doesn't go unnoticed. I think it's fine even if it's not removed

Reply:
Awesome. Definitely a good thing to point out. I think it makes sense to try to make this process as automated as possible.

It took me a little longer this time because I ended up having to add a couple languages.